### ISSUE TITLE
Night Light not working on fresh Regolith install

### ISSUE THREAD

**Issue by p3k** [regolith-linux/regolith-desktop#609]
- [x] I've [run a search on the existing bug reports](https://help.github.com/en/github/searching-for-information-on-github/searching-issues-and-pull-requests) to make sure I'm not filing a duplicate (you can use the search box at the top left corner)
- [x] I've read [the Regolith documentation](https://regolith-linux.org/docs/) and didn't find any relevant topic
- [ ] I tried [asking on Slack](https://regolith-linux.org/interact/) (especially for configuration related issues)

**Describe the bug**
I upgraded Ubuntu 18.04 to 20.04, added the PPA and installed Regolith. Nothing happens when I change the slider in the Night Light settings.

**To Reproduce**
Go to the Night Light settings, enable it if necessary, and change the color temperature.

![Screenshot from 2021-09-08 13-55-49](https://user-images.githubusercontent.com/16631/132505015-d45c8150-3a71-4a27-bd16-527de5924f9b.png)

**Expected behavior**
The color temperature of the monitors should change accordingly.

**Screenshots**

Monitors are detected correctly.

![Screenshot from 2021-09-08 13-58-21](https://user-images.githubusercontent.com/16631/132505322-a21ec53c-b144-4d4e-a8f7-8936d0aa06fd.png)

**Configuration file(s)**

Did not configure anything related to Regolith, yet.

**Installation Details**
 - Regolith Install Type: PPA
 - Regolith Version: 1.6
 - PPA url: ppa:regolith-linux/release
 - Host OS (for PPA): Ubuntu 20.04

**Additional context**

When I drag the slider these lines appear in `/var/log/syslog`:

```plain
Sep  8 13:51:18 lichte gsd-color[14429]: could not find device: property match 'XRANDR_name'='
DVI-D-1' does not exist
Sep  8 13:51:18 lichte gsd-color[14429]: could not find device: property match 'XRANDR_name'='
HDMI-1' does not exist
```

However, `xrandr` does output exactly these IDs:

```plain
→ xrandr --listactivemonitors 
Monitors: 2
 0: +*HDMI-1 1920/518x1200/324+0+0  HDMI-1
 1: +DVI-D-1 1920/518x1200/324+1920+0  DVI-D-1
```

**Comment by p3k** [comment]
Killing `gsd-color` has no effect, it just is restarted, anyway.

**Comment by kgilmer** [comment]
I have verified this but found that if I log out and back in again that the color settings suddenly work.  Did you log out after making this change and see any difference @p3k ?

**Comment by p3k** [comment]
need to check this when i am back at the machine (office)…

**Comment by p3k** [comment]
nope, not even after a powered-off weekend and fresh start this morning the night light works.

**Comment by amireldor** [comment]
Hello persons, I experience this issue or something related. I'm back on Regolith after a while. I installed a fresh Ubuntu 21.10 and then used the PPA to install Regolith.

I had the night light enabled on GNOME before I installed Regolith. When I log in under the Regolith session, there's no night light, even though it's enabled in Settings (I go there with Super+C). When I disable it, it seems like the screen is night-lighted instantly and then "fades" back to normal light. When I re-enable the night light after that, it kicks in and seem to work fine (I don't know if it goes back to normal light at sunrise, I'm sleeping at that time :smiling_face_with_tear: ).

**Comment by cwh1te** [comment]
I had this problem with similar gsd-color errors in syslog, but after killing the gsd-color process with `kill -4 <pid>` I was able to turn on the night light.

**Comment by mpareja** [comment]
I can confirm that `pkill gsd-color` allows night light mode to kick in.

**Comment by Giom-fm** [comment]
Hello,

unfortunately killing gsd-color does nothing for me.
However, I noticed that for `org.gnome.SettingsDaemon.Color.target` crashes with the following error message: 

`org.gnome.SettingsDaemon.Color.target: Job org.gnome.SettingsDaemon.Color.target/start failed with result 'dependency'`.

Is this the case for you too or is the service running?

**Comment by mpareja** [comment]
> unfortunately killing gsd-color does nothing for me.

Yes, this trick is no longer working for me on newer versions of Ubuntu.

**Comment by mimre25** [comment]
I experience the same issue just like [amireldor](https://github.com/amireldor) describes. 

When I log in, the night light doesn't work.
Then I go to the settings, turn it off ~ it flicks on and fades off, and then I turn it on and it works fine.

I'm still on Ubuntu 20.04 but with regolith 3:
<details>

<summary>➜  ~ apt show regolith-desktop</summary>

```sh
Package: regolith-desktop
Version: 4.4.2-1regolith-focal
Priority: optional
Section: x11
Maintainer: Ken Gilmer <kgilmer@gmail.com>
Installed-Size: 16,4 kB
Provides: regolith-desktop-meta
Depends: regolith-desktop-session
Suggests: update-manager, software-properties-gtk
Conflicts: regolith-desktop-meta, unclutter-startup
Replaces: regolith-desktop-meta
Homepage: https://github.com/regolith-linux/regolith-desktop
Download-Size: 5.556 B
APT-Manual-Installed: yes
APT-Sources: https://regolith-desktop.org/release-3_0-ubuntu-focal-amd64 focal/main amd64 Packages
Description: Meta package for Regolith desktop environment
 This package specifies the root package from which the Regolith
 desktop environment can be installed.
````


</details>

**Comment by peterrus** [comment]
Workaround: https://github.com/orgs/regolith-linux/discussions/797#discussioncomment-9725952
